prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <div> <a href="home.html"> click to go to the home page </a></div>
```

Comment:
You can get back to the `index.html` page with a relative path.

`href="../index.html"`

The `../` means to "go up one directory" before looking for `index.html`.